cmd/cluster/detachstuckvolume.go

```diff
@@ -0,0 +1,229 @@
+	//Region   []string
+	VolumeId []string
+}
+
+type detachStuckVolumeOptions struct {
+	clusterID string
+	cluster   *cmv1.Cluster
+}
+
+func newCmdDetachStuckVolume() *cobra.Command {
+	ops := newdetachStuckVolumeOptions()
+	detachstuckvolumeCmd := &cobra.Command{
+		Use:               "detach-stuck-volume",
+		Short:             "Detach openshift-monitoring namespace's volume from a cluster forcefully",
+		Args:              cobra.NoArgs,
+		DisableAutoGenTag: true,
+		Run: func(cmd *cobra.Command, args []string) {
+			cmdutil.CheckErr(ops.complete(cmd, args))
+			cmdutil.CheckErr(ops.run())
+		},
+	}
+	detachstuckvolumeCmd.Flags().StringVarP(&ops.clusterID, "cluster-id", "c", "", "The internal ID of the cluster to perform actions on")
+	detachstuckvolumeCmd.MarkFlagRequired("cluster-id")
+	return detachstuckvolumeCmd
+
+}
+
+func newdetachStuckVolumeOptions() *detachStuckVolumeOptions {
+	return &detachStuckVolumeOptions{}
+}
+
+func (o *detachStuckVolumeOptions) complete(cmd *cobra.Command, args []string) error {
+
+	err := utils.IsValidClusterKey(o.clusterID)
+	if err != nil {
+		return err
+	}
+	connection, err := utils.CreateConnection()
+	if err != nil {
+		return err
+	}
+	defer connection.Close()
+	cluster, err := utils.GetCluster(connection, o.clusterID)
+	if err != nil {
+		return err
+	}
+	o.cluster = cluster
+	o.clusterID = cluster.ID()
+	if strings.ToUpper(cluster.CloudProvider().ID()) != "AWS" {
+		return errors.New("this command is only available for AWS clusters")
+	}
+
+	return nil
+}
+
+func (o *detachStuckVolumeOptions) run() error {
+
+	_, _, clientset, err := getKubeConfig(o.clusterID)
+
+	if err != nil {
+		return fmt.Errorf("failed to retrieve Kubernetes configuration and client for cluster with ID %s: %w", o.clusterID, err)
+	}
+
+	err = volIdRegion(clientset, Namespace, "")
+	if err != nil {
+		return err
+	}
+
+	// If the volIdRegion found no pv is utilized by non running state pod. Which make global variable nil.
+	// Thus there's a panic exit. In order to prevent it we're using following logic to prevent panic exit.
+	if len(detachStuckVolumeInput.VolumeId) == 0 {
+		return fmt.Errorf("there's no pv utilized by non running state pod in cluster: %s\nNo action required", o.clusterID)
+	}
+
+	/*
+		if len(detachStuckVolumeInput.Region) != 1 {
+			return fmt.Errorf("got more than one region value: %v", len(detachStuckVolumeInput.Region))
+		}
+	*/
+
+	//fmt.Println(detachStuckVolumeInput.Region[0])
+
+	fmt.Printf("The volume id are %v\n", detachStuckVolumeInput.VolumeId)
+
+	// aws ec2 detach-volume --volume-id $VOLUME_ID --region $REGION --force
+	// WiP - Need to convert above cmd to function once volIdRegion gets completed
+	// Tested till line 107 - Couldn't test below aws function getting priv issue. gig acc doesn't have nessary priv
+
+	ocmClient, err := utils.CreateConnection()
+	if err != nil {
+		return err
+	}
+	defer ocmClient.Close()
```

Comment:
We already have a client intially in the `complete` function
Instead of starting a new one, we can use the exisitng one for the entire command